app/models/app.rb

```diff
@@ -8,6 +8,8 @@ class App < ApplicationRecord
   scope :published, -> { where(published_status: 'PUBLISHED')}
 
   has_many :search_documents, :class_name => 'Document', as: :searchable
+  has_one :categorization, :as => :categorizationable
```

Comment:
app moze byt len v jednej kategorii? Preco?

Reply:
Aha, som zmateny z tychto nazvov. Ak nieco moze byt zakategorizovane tak to je categorizable (to by mal byt asi ten model - co ma polymorphic na rozne "nodes"). Nieco nam brani to nazvat `Categorizable` kde bude tabulka `categorizables`?

Reply:
Za tým názvom bola taká myšlienka, že medzi `journey` a `category` existuje model `categorization`, pričom `categorization` je ako keby `categorizable` pomocou `category`. Preto som to chcel významovo odlíšiť, že `journey` (a ďalšie 2) vedia byť `categorizationable` pomocou `categorization`.

Ale keď na to teraz tak pozerám, tak to v pohode môžeme nazvať `categorizable` :smile:

Reply:
Ok, ale model a tabuľku nemôžeme nazvať `-able`. To bude `Categorization` a v tom polymorfizme sa bude volať `categorizable`. Tak to vraj ruby chce a nechcem to tam moc miešat.